src/modules/account/components/AccountRow/AccountRow.js

```diff
@@ -18,9 +20,21 @@ function AccountRow({ account, onSelect, onRemove, truncate }) {
     >
       <WalletVisual address={address} size={40} />
       <div className={styles.addressWrapper}>
-        <b className={`${styles.addressValue}`}>
-          <span>{name}</span> {isHW && <Icon name="hardwareWalletIcon" />}
-        </b>
+        <div className={styles.header}>
+          <b className={`${styles.addressValue}`}>
+            <span>{name}</span> {isHW && <Icon name="hardwareWalletIcon" />}
+          </b>
+          {!(isHW && isNew) ? null : (<div>
```

Comment:
Couldnt we just use `&&` 
`(isHW && isNew) && (<div ...` 
instead of using `? null` with negation.

Also, file should be prettified.